proto/beacon/p2p/v1/messages.pb.go

```diff
@@ -5,9 +5,10 @@ package ethereum_beacon_p2p_v1
 
 import (
 	fmt "fmt"
-	proto "github.com/gogo/protobuf/proto"
```

Comment:
Does your IDE do this? Can you tell it not to do it? This is a generated file and anyone that runs the script update will revert all of these changes.

Reply:
@prestonvanloon I just ran goimports and it updated this

Reply:
Looks like it was fixed here: https://github.com/golang/protobuf/pull/719
I'll see what we can do to get this into our repo

Reply:
shall I revert?